src/utils/enums.ts

```diff
@@ -0,0 +1,17 @@
+export enum FilterForTodos {
```

Comment:
its better to think about vore clear name like `TodoFilterType` or something like that